src/NosCore.Handler/CommandPacketHandler.cs

```diff
@@ -29,13 +29,9 @@ public CommandPacketHandler(ClientSession session)
         #region Methods
         public void Speed(SpeedPacket speedPacket)
         {
-            if (speedPacket.Speed >= 60)
-            {
-                return;
-            }
-            Session.Character.Speed = speedPacket.Speed;
+            Session.Character.Speed = (speedPacket.Speed >= Convert.ToByte(60) ? Convert.ToByte(59) : speedPacket.Speed);
```

Comment:
a cast to byte here would be better, shouldn't be nescessary you can check if Speed and packet.speed are bytes (they should be)